### ISSUE TITLE
Prepare a template for a job advertisement

### ISSUE THREAD

**Issue by chriskodehub** [chriskodehub/kodehode-course-plan#2]
The template should be divided into four elements

1. Title of the position
2. Responsibilities
3. Requirements
4. Method of contact

The goal is to train the soft skills that we learn during the course and get used to stress.

**Comment by RaJoBj** [comment]
👍

**Comment by chriskodehub** [comment]
We're not closing yet 😁 I'll send you a template proposal in a moment.

**Comment by chriskodehub** [comment]
# Junior Software Developer Job

### Employer

Kodehode

### Start date

Immediately

### Job type

Adapted to the candidate's abilities and remote

###  Job Description

We are looking for an enthusiastic junior software developer to join our experienced software design team. You will report directly to the development manager and assist with all software coding and design functions. Your primary focus will be learning the codebase, gathering user data, and responding to senior developers' requests.

To ensure success as a junior software developer, you should have an excellent working knowledge of primary programming languages, learn new technology quickly, and work in a team environment. Ultimately, a top-class Junior Software Developer provides valuable support to the design team while continually improving their coding and design skills.

### Junior Software Developer Responsibilities

- Assisting the development manager with all aspects of software design and coding.
- Attending and contributing to company development meetings.
- Learning the codebase and improving your coding skills.
- Writing and maintaining code.
- Working on minor bug fixes.
- Monitoring the technical performance of internal systems.
- Responding to requests from the development team.
- Gathering information from consumers about program functionality.
- Writing reports.
- Conducting development tests

### Junior Software Developer Requirements

- A minimum of 3 days of study on the Kodehode course
- Bachelor's degree in computer science.
- Knowledge of primary coding languages, including C++, HTML5, and JavaScript.
- Basic programming experience.
- Knowledge of databases and operating systems.
- Good working knowledge of email systems and Microsoft Office software.
- Ability to learn new software and technologies quickly.
- Ability to follow instructions and work in a team environment.
- Detail-oriented.

### Junior Software Developer Benefits

- Interesting and inspiring tasks in a real-life project
- The possibility of practising the acquired skills
- Learning to work in a team and training soft skills
- The possibility of a quick promotion
- Possibility to include information about participation in the project in your CV and a link to the project

**Comment by chriskodehub** [comment]
Of course, we will adapt the requirements and scope of skills to a given position.

**Comment by RaJoBj** [comment]
Hei!
Her er mine innspill:
Employer: Kodehode Tønsberg.
Teksten er litt lang, jeg ville kort beskrevet prosjektet og så spurt om det er noen som ønsker å bidra, slik du har gjort i den foreløpige prosjektbeskrivelsen.
Annonsen er god hvis vi tenker en stilling over lengre tid, men her har dere liten tid og trenger å få entusiastene til å bidra.

**Comment by chriskodehub** [comment]
# Junior Software Developer Job

### Employer

Kodehode Tønsberg

### Start date

Immediately

### Job type

Adapted to the candidate's abilities and remote

###  Job Description

The idea is simple. We want to make a practical application that Kodehode can use to make less work of making our weekly learning plans. The idea is exciting, and most importantly, we will be able to develop our skills and make a practical application.

Who is willing to help?

### Junior Software Developer Responsibilities

- Creating a good atmosphere
- Collaboration with other developers

### Junior Software Developer Requirements

- A minimum of 1 day of study on the Kodehode course 😁
- Ability to learn new software and technologies quickly.
- Ability to follow instructions and work in a team environment.
- Detail-oriented.

### Junior Software Developer Benefits

- Interesting and inspiring tasks in a real-life project
- The possibility of a quick promotion
- Possibility to include information about participation in the project in your CV and a link to the project 💪

### How to apply

Send your CV if you have one. You can also write a few words about yourself.

---

@RaJoBj I don't know if you know about it, but the `Close` button is used only when a given task is finished and the thread's author does it. When you click `Close`, the message disappears from the list and is hidden.

**Comment by RaJoBj** [comment]
👍

**Comment by RaJoBj** [comment]
Legger du det ut på Discord?

**Comment by chriskodehub** [comment]
Ikke ennå, Sine ba om en handlingsplan. Jeg må sende den til henne i morgen. Jeg venter på hva han sier. Hvis prosjektet blir godkjent, vil jeg utarbeide noen få konkrete stillinger og sende dem til Discord mandag morgen. Det er Mandags kickoff på mandag morgen, så kanskje lærerne kan nevne dette prosjektet?

**Comment by chriskodehub** [comment]
Nå jobber jeg med en veiledning for alle, "Hvordan komme i gang?".
Dette er en ny idé for alle, og jeg vil at alle skal føle seg vel og vite hva og hvor de kan gjøre.
